src/cmd/build.js

```diff
@@ -56,8 +100,19 @@ async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
+  let messageFile: string;
+  let extensionName: string = manifestData.name;
+
+  if (manifestData.default_locale) {
+    messageFile = path.join(sourceDir, '_locales',
+                                    manifestData.default_locale,
+                                    'messages.json');
+    log.info(messageFile);
+    extensionName = await getDefaultLocalizedName
+                                      ({messageFile, manifestData});
```

Comment:
always break lines after an opening parenthesis (not before), like this:
````
getDefaultLocalizedName(
  {messageFile, manifestData});
````
This helps the reader follow the nesting easier